From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Mon, 4 Oct 2021 15:28:37 -0700
Subject: [PATCH] Disable config from redhat-rpm-config

This file appears to originate from `redhat-rpm-config`, which is not
available in Clear Linux OS.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 build-aux/ltmain.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build-aux/ltmain.sh b/build-aux/ltmain.sh
index e9c06d7..0f0a2da 100644
--- a/build-aux/ltmain.sh
+++ b/build-aux/ltmain.sh
@@ -6523,7 +6523,7 @@ func_mode_link ()
     old_convenience=
     deplibs=
     old_deplibs=
-    compiler_flags="-specs=/usr/lib/rpm/redhat/redhat-hardened-ld"
+    compiler_flags=
     linker_flags=
     dllsearchpath=
     lib_search_path=`pwd`
